src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -454,6 +454,16 @@ export default class GroupComparisonStore extends ComparisonStore {
         },
     });
 
+    readonly genesWithMutations = remoteData<Gene[]>({
+        await: () => [this.genesWithHighestMutationFrequency],
+        invoke: async () => {
+            const genes = await fetchGenes(
```

Comment:
seems odd that we are fetching these genes here when we already fetching all genes right above in availableGenes.  could we not just filter that here?

Reply:
also, seems strange that genesWithMutations is produced from genesWithHighestMutationFrequency.  That would imply the it wasn't just "highest" but all those with ANY mutation frequency, perhaps sorted by frequency?

Reply:
yea we don't need to fetch genes here since we already have availableGenes, fixed. and yea genesWithHighestMutationFrequency is not the best name, changed it to genesSortedByMutationFrequency.